server/segment_rpc.go

```diff
@@ -508,6 +508,12 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 		return "", nil
 	}
 
+	// Compare Orchestrator Price against BroadcastConfig.MaxPrice
+	ok := validatePrice(sess)
```

Comment:
We can omit the `ok` variable here and instead do:

```
if !validatePrice(sess) {
    ...
}
```

Alternatively, if `validatePrice()` returns an error, as suggested in my other comment, instead of a boolean we can do:

```
if err := validatePrice(sess); err != nil {
    return "", err
}
```

Reply:
addressed in https://github.com/livepeer/go-livepeer/pull/992/commits/18648d78c0d532e7e1cc26d8bc1e374a560cee34